### PR TITLE
Add terradoc

### PR DESCRIPTION
**Pull request by kassianh**
Recreated README with terradoc, checked documentation for errors or inconsistencies.

### CODE REVIEW

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -0,0 +1,517 @@
+header {
+  image = "https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg"
+  url   = "https://mineiros.io/?ref=terraform-module-template"
```
Comment (soerenmartius) [inline README.tfdoc.hcl:3]: `terraform-module-template` should be replaced with  `terraform-google-network-subnet`

seems like you forgot to run `make template/adjust` ? :)

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: README.tfdoc.hcl [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+
+        The created subnet resources.
+    END
+  }
+
+  section {
+    title = "External Documentation"
+
+    section {
+      title   = "Google Documentation :"
```
Comment (mariux) [inline README.tfdoc.hcl:357]:
```suggestion
      title   = "Google Documentation"
```

File: README.tfdoc.hcl [hunk 3]
```diff
@@ -0,0 +1,517 @@
...
+      content = <<-END
+        - Configuring Private Google Access: <https://cloud.google.com/vpc/docs/configure-private-google-access>
+        - Using VPC networks: <https://cloud.google.com/vpc/docs/using-vpc>
+      END
+    }
+
+    section {
+      title   = "Terraform Google Provider Documentation :"
```
Comment (mariux) [inline README.tfdoc.hcl:365]:
```suggestion
      title   = "Terraform Google Provider Documentation"
```

**Review by mariux — DISMISSED** [review]

**Review by soerenmartius — APPROVED** [review]
lgtm

**Pull request merged**